docs/src/main/sphinx/release/release-439.md

```diff
@@ -0,0 +1,42 @@
+# Release 439 (14 Feb 2024)
+
+## General
+
+* Fix failure when setting session properties for a catalog with a `.` in its
+  name. ({issue}`20474`)
+
+## BigQuery connector
+
+* Fix failure when pushing down predicates onto BigQuery views. ({issue}`20627`)
+
+## Delta Lake connector
+
+* Improve performance of table scans by adding support for
+  [caching data on local storage](/connector/filesystem-cache). ({issue}`18719`)
+* Fix potential crash when decompressing corrupted Snappy data. ({issue}`20631`)
+
+## Hive connector
+
+* Improve performance of table scans by updating support for
+  [caching data on local storage](/connector/filesystem-cache). ({issue}`20658`, {issue}`20102`)
+* Fix query failure when a value has not been specified for the
+  `orc_bloom_filter_fpp` table property. ({issue}`16589`)
+* Fix potential query failure when writing to ORC files. ({issue}`20587`)
+* Fix potential crash when decompressing corrupted Snappy data. ({issue}`20631`)
```

Comment:
Dont know why but "decompressing" sounds odd in this sentence

Reply:
We could replace with "when reading corrupted Snappy data". From a user perspective, there's no difference.